PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -380,6 +404,20 @@
             image.imageAsset?.value(forKey: "_containingBundle") != nil
         }
 
+        // Photo library images have a UUID identifier as _assetName (e.g 64EF5A48-2E96-4AB2-A79B-AAB7E9116E3D)
+        // SF symbol and bundle images have the actual symbol name as _assetName (e.g chevron.backward)
```

Comment:
I wonder if its the same for all OS versions, and if it works for SwiftUI and UIKit?

Reply:
Well there's a similar approach in isAssetsImage so that's fine I guess